src/public/app/core/types/ScPlayer.ts

```diff
@@ -9,20 +9,20 @@ export interface ScPlayer {
 }
 
 export enum ScPlayerEvent {
-    StateChange = 'state-change',
-    Play = 'play',
-    PlayStart = 'play-start',
-    PlayResume = 'play-resume',
-    Pause = 'pause',
-    Finished = 'finish',
-    Seek = 'seek',
-    Seeked = 'seeked',
-    GeoBlocked = 'geo_blocked',
-    BufferingStart = 'buffering_start',
-    BufferingEnd = 'buffering_end',
-    AudioError = 'audio_error',
-    Time = 'time',
-    NoStreams = 'no_streams',
-    NoProtocol = 'no_protocol',
-    NoConnection = 'no_connection'
+    StateChange = <any>'state-change',
```

Comment:
Or just scrap the strings altogether. Is there a reason you added strings to go along? Why not just
enum { StateChange, Play, PlayStart, ... }